Helper/Webhook.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Klaviyo\Reclaim\Helper;
+
+use Exception;
+use \Klaviyo\Reclaim\Helper\ScopeSetting;
+
+class Webhook extends \Magento\Framework\App\Helper\AbstractHelper
+{
+    const USER_AGENT = 'Klaviyo/Webhook/1.0';
```

Comment:
is this going to be the standard for all extensions?  Do you think we should also contain the name of the integration within it?  Like `Klaviyo/MagentoTwo/Webhook/1.0` ?